### PR TITLE
BitMEX issues p2wsh addresses only

### PR DESCRIPTION
**Pull request by instagibbs**
Circa early 2021

source: I deployed it

### CODE REVIEW

File: _data/compatibility/bitmex.yaml [hunk 1]
```diff
@@ -34,11 +34,11 @@ segwit:
...
       p2sh_wrapped: "false"
-      bech32: "false"
-      default: "p2sh"
+      bech32: "true"
+      default: "bech32"
     send:
       bech32: "true"   # https://blog.bitmex.com/bitmex-enables-bech32-sending-support/
-      change_bech32: "false"
+      change_bech32: "true"
       segwit_v1: "Bech32 not supported."
```
Comment (Xekyo) [inline _data/compatibility/bitmex.yaml:42]: According to this, BitMEX should also support bech32m now: https://twitter.com/BitMEXResearch/status/1492152557044654082

@bitschmidty: What's the story behind the `segwit_v1` fields? We don't show those yet, do we?

**Review by Xekyo — COMMENTED** [review]
Looks good, but  perhaps also touch up the `segwit_v1` field

File: _data/compatibility/bitmex.yaml (hunk 1 above)
Comment (instagibbs) [reply _data/compatibility/bitmex.yaml:42]: ah yeah that's true as well

**Comment by bitschmidty** [comment]
Thanks @instagibbs !

**Comment by shuckc** [comment]
Thanks @instagibbs

**Pull request merged**